packages/cra-template-edu/template.json

```diff
@@ -0,0 +1,47 @@
+{
+  "package": {
+    "scripts": {
+      "build-storybook": "build-storybook -s public",
+      "lint": "eslint --ext=js,jsx,ts,tsx .",
+      "storybook": "start-storybook -p 6006 -s public",
+      "storybook:axe": "build-storybook -s public && axe-storybook",
+      "types": "tsc --noEmit"
+    },
+    "dependencies": {
+      "@chanzuckerberg/eds": "^5.3.1",
```

Comment:
We'll probably want to keep all these deps relatively up to date.

Couple ideas:
- FE infra oncall updates and publishes a new version every week or month?
- When publishing a new EDS major version, also update here?

Reply:
> FE infra oncall updates and publishes a new version every month?

Yeah, that seems like a good idea to me. If we wanted to, we could try using Renovate to create the monthly PR. A developer would still have to review/approve and then publish the new version though.

Reply:
Good idea 👍 

One challenge may be that this file has a slightly different structure than a normal package.json (just nested under a "package" key).

But it would be great if Renovate works for this use case